src/shared/src/components/LandmarkShortcuts.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface ILandmarkShortcutsProps {
+  navToMainText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  'sr-only': {
+    position: 'absolute',
+    width: '1px',
+    height: '1px',
+    padding: 0,
+    margin: '-1px',
+    overflow: 'hidden',
+    clip: 'rect(0,0,0,0)',
+    border: 0,
+  },
+  'sr-only-focusable': {
+    '&:focus': {
+      position: 'relative',
```

Comment:
Should the entire page jump, or is the intent to only have the button appear as a layover.
Maybe position: fixed and top: 0px, left: 0px etc. could work?

Reply:
Entire page, as we want consistency across solutions.